File: styler-service/src/main/java/com/lookgen/styler/repo/SketchRepository.java
package com.lookgen.styler.repo;

import com.lookgen.styler.domain.Sketch;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface SketchRepository extends JpaRepository<Sketch, Long> {
    Sketch findByUserId(Long userId);
}

File: styler-service/src/main/kotlin/com/lookgen/styler/repo/SketchRepository.kt
package com.lookgen.styler.repo

import com.lookgen.styler.domain.Sketch
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface SketchRepository : JpaRepository<Sketch, Long> {
    fun findByUserId(userId: Long): Sketch?
}
